graspologic/simulations/simulations.py

```diff
@@ -966,15 +966,12 @@ def mmsbm(
         arr=mm_vectors,
     )
 
-    P = np.zeros((n, n))
-    for i in range(n):
-        for j in range(n):
-            P[i, j] = p[labels[i, j], labels[j, i]]
+    P = p[(labels, labels.T)]
```

Comment:
does this work as we expected?

Reply:
Apologies for the late reply to the comment!

Yes, it does. I ran the two different codes in a notebook and the used np.allclose to check that the P matrices generated are identical